lib/rules/no-restricted-browser-globals-during-ssr.js

```diff
@@ -0,0 +1,843 @@
+    'Selection',
+    'ShadowRoot',
+    'SharedWorker',
+    'SourceBuffer',
+    'SourceBufferList',
+    'SpeechSynthesisErrorEvent',
+    'SpeechSynthesisEvent',
+    'SpeechSynthesisUtterance',
+    'StaticRange',
+    'StereoPannerNode',
+    'Storage',
+    'StorageEvent',
+    'StylePropertyMap',
+    'StylePropertyMapReadOnly',
+    'StyleSheet',
+    'StyleSheetList',
+    'SubmitEvent',
+    'SyncManager',
+    'TaskAttributionTiming',
+    'TaskController',
+    'TaskPriorityChangeEvent',
+    'TaskSignal',
+    'Text',
+    'TextDecoderStream',
+    'TextEncoderStream',
+    'TextEvent',
+    'TextMetrics',
+    'TextTrack',
+    'TextTrackCue',
+    'TextTrackCueList',
+    'TextTrackList',
+    'TimeRanges',
+    'Touch',
+    'TouchEvent',
+    'TouchList',
+    'TrackEvent',
+    'TransformStream',
+    'TransformStreamDefaultController',
+    'TransitionEvent',
+    'TreeWalker',
+    'TrustedHTML',
+    'TrustedScript',
+    'TrustedScriptURL',
+    'TrustedTypePolicy',
+    'TrustedTypePolicyFactory',
+    'UIEvent',
+    'URLPattern',
+    'UserActivation',
+    'VTTCue',
+    'ValidityState',
+    'VideoColorSpace',
+    'VideoFrame',
+    'VideoPlaybackQuality',
+    'VirtualKeyboardGeometryChangeEvent',
+    'VisualViewport',
+    'WaveShaperNode',
+    'WebGL2RenderingContext',
+    'WebGLActiveInfo',
+    'WebGLBuffer',
+    'WebGLContextEvent',
+    'WebGLFramebuffer',
+    'WebGLProgram',
+    'WebGLQuery',
+    'WebGLRenderbuffer',
+    'WebGLRenderingContext',
+    'WebGLSampler',
+    'WebGLShader',
+    'WebGLShaderPrecisionFormat',
+    'WebGLSync',
+    'WebGLTexture',
+    'WebGLTransformFeedback',
+    'WebGLUniformLocation',
+    'WebGLVertexArrayObject',
+    'WebKitCSSMatrix',
+    'WebKitMutationObserver',
+    'WebSocket',
+    'WheelEvent',
+    'Window',
+    'WindowControlsOverlay',
+    'WindowControlsOverlayGeometryChangeEvent',
+    'Worker',
+    'WritableStream',
+    'WritableStreamDefaultController',
+    'WritableStreamDefaultWriter',
+    'XMLDocument',
+    'XMLHttpRequest',
+    'XMLHttpRequestEventTarget',
+    'XMLHttpRequestUpload',
+    'XMLSerializer',
+    'XPathEvaluator',
+    'XPathExpression',
+    'XPathResult',
+    'XSLTProcessor',
+    'alert',
+    'blur',
+    'cancelAnimationFrame',
+    'cancelIdleCallback',
+    'captureEvents',
+    'chrome',
+    'clientInformation',
+    // 'close',
+    // 'closed',
+    'confirm',
+    'createImageBitmap',
+    'crossOriginIsolated',
+    'customElements',
+    'devicePixelRatio',
+    'document',
+    // 'event',
+    // 'external',
+    // 'find',
+    // 'focus',
+    'frameElement',
+    'frames',
+    'getComputedStyle',
+    'getSelection',
+    'history',
+    'indexedDB',
+    'innerHeight',
+    'innerWidth',
+    'isSecureContext',
+    'launchQueue',
+    // 'length',
+    'localStorage',
+    'location',
+    'locationbar',
+    'matchMedia',
+    'menubar',
+    'moveBy',
+    'moveTo',
+    // 'name',
```

Comment:
I removed some very common identifiers like this from the list because they're used as variables.